README.adoc

```diff
@@ -201,37 +229,42 @@ You can build everything from the root of the repository.
 
 `./mvnw clean install`
 
-But, this may not be what you are interested in doing since you are probably interested in a single application or a few of them.
-In order to build the functions and applications that you are interested in, you need to build them selectively as shown below.
+However, this may not be what you are interested in since you are probably interested in a single application or a few of them.
+
+To build the functions and applications that you are interested in, you need to build them selectively, as shown below.
 
-==== Building functions
+==== Building Functions
 
 `./mvnw clean install -f functions`
 
 You can also build a single function or group of functions.
-For e.g if you are only interested in jdbc-supplier and log-consumer, do the following.
+For example, if you are only interested in `jdbc-supplier` and `log-consumer`, do the following.
```

Comment:
So, yes, if we call this as "`jdbc-supplier`", then "`time-source`" is better name to use above.